_validate/addonVersion_schema.json

```
@@ -124,7 +126,7 @@
 				"Makes doing XYZ easier"
 			],
 			"title": "The description (en) of the addon",
-			"type": "string"
+			"type": ["string", "null"]
```

this is a required value

```suggestion
			"type": "string"
```